ci/test

```diff
@@ -42,7 +42,7 @@ function run_cucumber_tests() {
   profile="$1"
 
   # Generate a data key
-  export CONJUR_DATA_KEY="$(openssl rand -base64 32)"
+  CONJUR_DATA_KEY="$(openssl rand -base64 32)"
```

Comment:
If you don't `export` this, it does nothing, because it's not used anywhere else in this file. Better to just remove it altogether.

Reply:
So, yes, this change would have fixed the problem of having each cucumber run get a different `CONJUR_DATA_KEY`. *None* of them would be using the one set here. :)

Unless I'm missing something, it should go.